tools/DfmHttpService/DfmHttpServer.cs

```diff
@@ -35,6 +35,7 @@ public void Start()
 
             _listener.Start();
             _processing.Reset();
+            Console.WriteLine("Ready");
```

Comment:
This is not a log, it is a output to tell extension that server is ready.

---
In reply to: [108107907](https://github.com/dotnet/docfx/pull/1490#discussion_r108107907) [](ancestors = 108107907)